doc/userguide/rules/intro.rst

```diff
@@ -228,10 +228,15 @@ Direction
 
 The directional arrow indicates which way the signature will be evaluated.
 In most signatures an arrow to the right (``->``) is used. This means that only
-packets with the same direction can match. However, it is also possible to
-have a rule match both directions (``<>``)::
+packets with the same direction can match.
+It is also possible to have a double arrow (``=>``) which means that the
+directionality for adresses and ports is used,
+but such a rule can match a bidirectional transaction, using keywords
+matching in each direction.
```

Comment:
> would it be possible to say that using => has a similar effect as having two rules (one for each side of the bidir transaction) and setting flowbits to only have an alert in case both match?

It is the goal indeed cf tests/detect-bidir-flowbits/ in https://github.com/OISF/suricata-verify/pull/1922

The goal is to be better as these are txbits, not flowbits ;-)